.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: "Run Unit-Tests"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
```

Comment:
can you add name for unit-tests like `name:Run Unit Tests` etc

Reply:
will do!